src/main/kotlin/org/jmailen/gradle/kotlinter/tasks/InstallPrePushHookTask.kt

```diff
@@ -0,0 +1,91 @@
+package org.jmailen.gradle.kotlinter.tasks
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+import org.jmailen.gradle.kotlinter.support.findGitDir
+import org.jmailen.gradle.kotlinter.support.findGradlew
+import java.io.File
+
+/**
+ * Install or update the kotlinter-gradle pre-push hook.
+ */
+open class InstallPrePushHookTask : DefaultTask() {
+    @TaskAction
+    fun run() {
+        val dotGitDir = findGitDir(project.rootDir)
+        logger.info(".git directory: $dotGitDir")
```

Comment:
For simplicity to the user, probably sufficient at the end to info level output either `created .git/hooks/pre-push` or `updated .git/hooks/pre-push`

Reply:
How about making the final message a `quiet` and these earlier ones `info`?

Reply:
The user is only likely to see `quiet`. If you think there's useful troubleshooting info you'd ask for if someone is having trouble, I'd include it in `debug` level. We do that elsewhere in the plugin and it did help in a couple of issues filed.